ui/app/src/Utils/Views.elm

```diff
@@ -131,6 +134,56 @@ validatedField htmlField labelText classes inputMsg blurMsg field =
                 ]
 
 
+validatedTextareaField : (List (Attribute msg) -> List (Html msg) -> Html msg) -> String -> String -> (String -> msg) -> msg -> ValidatedField -> Html msg
+validatedTextareaField htmlField labelText classes inputMsg blurMsg field =
+    let
+        lineCount =
+            lines field.value
```

Comment:
`String` is always imported, so you can write `String.lines` here and remove `import String exposing (lines)` from the top

Reply:
Sorry, lack of elm knowledge.
Thank's for Review.

Reply:
No worries, thanks for working on this!